### ISSUE TITLE
Momentum distance slowdown

### ISSUE THREAD

**Issue by jtangelder** [forcedotcom/scrollerjs#4]
Sometimes the velocity/distance calculation is a bit off scale, and the scroller just shoots out and goes nuts! Some kind of slowdown math would fix this... I thought it would be an issue with a too high velocity value, but the value looks correct to me!

Must be something at https://github.com/forcedotcom/scrollerjs/blob/master/src/scroller.js#L1107

I guess you've noticed this too, but just want to mention it. :+1:

**Comment by karneaud** [comment]
Bump!

**Comment by diervo** [comment]
I'm not sure what is exactly the problem to be able to fix it. Any example will help (or video)

**Comment by netgfx** [comment]
yes this is indeed an issue, it would help if we could define a max-velocity property, I understand this might be against the formula of the gesture calculation but too fast scrolling is breaking user experience (user has to be able to read what he is scrolling through)
